pkg/minikube/detect/detect.go

```diff
@@ -107,3 +107,9 @@ func DockerInstalledViaSnap() bool {
 
 	return strings.Contains(string(o), "snap")
 }
+
+// GithubActionRunner returns true if running inside a github action runner
+func GithubActionRunner() bool {
```

Comment:
This func should be removed from the test file you copied it from and the tests that reference it should be updated to use this one.